src/radarChart.ts

```diff
@@ -1002,17 +1000,18 @@ module powerbi.extensibility.visual {
                 .attr("points", calculatePoints)
                 .attr("points-count", (dataPoints: RadarChartDatapoint[]) => dataPoints.length);
 
+            let settings: RadarChartSettings = this.radarChartData.settings;
```

Comment:
Please increase version and cover changes in CHANGELOG if we will publish this fix